_inc/client/admin.js

```diff
@@ -85,6 +85,11 @@ function render() {
 						name={ i18n.translate( 'Plans', { context: 'Navigation item.' } ) }
 						component={ Main }
 					/>
+					<Route
+						path="/plans-prompt"
+						name={ i18n.translate( 'Plans Prompt', { context: 'Navigation item.' } ) }
```

Comment:
Maybe let's keep it "Plans".

```suggestion
						name={ i18n.translate( 'Plans', { context: 'Navigation item.' } ) }
```